pages/windows/choice.md

```diff
@@ -1,29 +1,28 @@
 # choice
 
-> Prompts the user to select one item from a list of single-character choices in a batch program, and then returns the index of the selected choice.
-> If used without parameters, choice displays the default choices Y and N.
+> Prompt user to select a choise and return the selected choice index.
 > More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/choice>.
 
-- A,B and C as list of choices to be used:
+- Prompt user to select `Y` or `N` choice:
```

Comment:
Not sure if I am correct or not, but i feel like we should have a `the` before the `user`:
```suggestion
- Prompt the user to select `Y` or `N` choice:
```

I think this comment, if correct should be applied to all IMO

Reply:
Done. I've used `the current user` instead of just `the user`.